README.md

```diff
@@ -227,4 +227,34 @@ Arguments:
 Options: 
     --help, -h -> Usage info
 ```
- 
+    
+The boolean property `strictSubcommandOptionsOrder` defines the allowed order of options and arguments for subcommands. 
+When it false (default), then the main program's options can be specified everywhere, even after the subcommand. 
+Otherwise, parameters can only be specified after the subcommands where they are defined. For example
+
+```kotlin
+@file:OptIn(ExperimentalCli::class)
+
+import kotlinx.cli.*
+
+fun main(args: Array<String>) {
+    val parser = ArgParser("example", strictSubcommandOptionsOrder = true)
+    val output by parser.option(ArgType.String, "output", "o", "Output file")
+
+    class Multiply: Subcommand("mul", "Multiply") {
```

Comment:
Let's add here also `strictSubcommandOptionsOrder = true` for consistency. I thought about propagation of this flag to all subcommands, but I decided this isn't good. So just make a test which can be used as an example with one setting for all subcommands.